lib/active_model/serializer/adapter.rb

```diff
@@ -16,16 +16,17 @@ def initialize(serializer, options = {})
         @options = options
       end
 
-      def serializable_hash(options = {})
+      def serializable_hash(options = nil)
         raise NotImplementedError, 'This is an abstract method. Should be implemented at the concrete adapter.'
```

Comment:
See https://github.com/bf4/active_model_serializers/commit/6ed30e3493a97f2232b9d2cda11abeae84a1e81e for why I changed the options to nil here and elsewhere